R/computeBAITpars.R

```diff
@@ -94,6 +97,7 @@ calcBAITpars <- function(model = "GFDL-ESM4",
     rast()
   # nolint end
 
+  terra::writeCDF(regPars, paste0("/p/tmp/hagento/output/", model, "/baitpars_", model, ".nc")) # nolint
```

Comment:
Either use the madrat output folder or a path given as an argument.

Reply:
This is only temporary and will be removed in the next PR.